source/agora/consensus/EnrollmentManager.d

```diff
@@ -61,6 +61,10 @@ public class EnrollmentManager
     /// Enrollment data object
     private Enrollment data;
 
+    /// The round period for revealing a preimage
+    /// It is an hour interval if a block is made in every 10 minutes
+    public static immutable uint PreimagePeriod = 6;
```

Comment:
We should make this configurable.

Reply:
I will make another issue now for configuring the period of revealing pre-images. Before the new issue gets completed, we should use that constant value.